From 7a22062425266ee55e04a58e9f74e0c7c2330824 Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Mon, 17 Nov 2014 20:58:47 -0500
Subject: [cpufreq] intel_pstate: Add CPUID for BDW-H CPU

Message-id: <20141117205847.35442.50988.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 100119
O-Subject: [PATCH RHEL7.1 BZ1164379] intel_pstate: Add CPUID for BDW-H CPU
Bugzilla: 1164379
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1164379

Description:
------------
Add BDW-H to the list of supported processors.

Signed-off-by: Dirk Brandewie <dirk.j.brandewie@intel.com>

Brew:
-----
http://brewweb.devel.redhat.com/brew/taskinfo?taskID=8254521

Upstream:
---------
http://marc.info/?l=linux-pm&m=141529625511510&w=2

Rafael J. Wysocki has acked and will push into 3.19. Posting now to make the beta cutoff.

Test Status:
------------
Intel is in the process of testing. If they find any problems I'll let
everyone know. Currently we do not have Broadwell H hardware yet.

---------------------------------------------------------------
Steve Best

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/intel_pstate.c b/drivers/cpufreq/intel_pstate.c
index c5eac94..e8d510b 100644
--- a/drivers/cpufreq/intel_pstate.c
+++ b/drivers/cpufreq/intel_pstate.c
@@ -660,6 +660,7 @@ static const struct x86_cpu_id intel_pstate_cpu_ids[] = {
  ICPU(0x3f, core_params),
  ICPU(0x45, core_params),
  ICPU(0x46, core_params),
+ ICPU(0x47, core_params),
  ICPU(0x4f, core_params),
  ICPU(0x56, core_params),
  {}
-- 
1.7.1